controllers/database_controller.go

```diff
@@ -190,7 +190,6 @@ func (r *DatabaseReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 		case dbPhaseCreate:
 			err := r.createDatabase(ctx, dbcr)
 			if err != nil {
-				logrus.Errorf("DB: namespace=%s, name=%s failed creating database - %s", dbcr.Namespace, dbcr.Name, err)
```

Comment:
delete error print because it will be printed here. https://github.com/kloeckner-i/db-operator/blob/cb7ff83fc9e4f9d62efe54e02d5b4fc424d42db4/controllers/database_controller.go#L750